packages/app-store/googlecalendar/components/ExistingGoogleCal.tsx

```diff
@@ -0,0 +1,54 @@
+import Link from "next/link";
```

Comment:
Decided to move this component out of the Meet package and into GCal since we are using it in multiple apps

Reply:
Considering we move the requiresApp to config.json, we should make this component generic.
If I just change the text it can be reused for Typeform app as well which requires Routing Forms
<img width="713" alt="Screenshot 2023-02-24 at 1 37 12 PM" src="https://user-images.githubusercontent.com/1780212/221125831-6805e024-6f9e-44a8-a8ce-cd202924ab51.png">

Reply:
yes i think the more generic the better. we will have the first app soon that requires Cal Video, too!